CONTRIBUTING.md

```diff
@@ -50,3 +50,15 @@ try to consolidate identical literals so that they are not duplicated. For QStri
 cannot be merged.
 
 ([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/))
+
+# Internationalization do's and don'ts
+
+Do:
+* enable strings visible in the Mudlet GUI to be translateable
+* minimise use of HTML styling tags in strings to be translated
+* enable users to use language-specific Mudlet object names (triggers, aliases, labels, etc)
+
+Don't:
+* translate the Mudlet API: functions, events, error messages or constants (`main` console, mapper directions)
+* use numbers in the API - English words are preferred instead
+* try to assemble a sentence on the fly - English grammar does not translate into other languages. Present the full sentence to translators instead
```

Comment:
* assume English-centric plural forms, other languages do not necessarily have the simple add an "s"/"es" for more/less then the singular case.†

* assume universal quote and number punctuation formats.  There are languages that use `«` and `»` instead of `"` for "quoting" words or phrases.  Qt can provide Locale specific displays of numbers/dates/times; we may have to look at them in more detail as we get further into Internationalization.

† Consider the `tr("Deleted %n rooms(s).", "", x )` form that takes the integer value `x` and puts it into the `%n` place-holder and allows multiple alternate sentence forms to be provided by translation as needed by the value of `x`.  This form is not that widely documented and only handles a phrase/sentence with a single variable so the source code may need revising to make use of this in cases where more than one value is involved.